### ISSUE TITLE
K6_DATADOG_TAG_BLACKLIST not functional

### ISSUE THREAD

**Issue by guongle-ssense** [loadimpact/k6#1602]
## Environment

- k6 version: 0.27.1
- OS and version: macOS Catalina
- Docker version and image, if applicable: 2.3.4.0 (469980)

## Expected Behavior

I am unsure how K6_DATADOG_TAG_BLACKLIST suppose to behave, but from several of the PRs I've checked within the repo, it is supposed to remove metrics that I would like the process to not send. However, I am getting an error regardless of what I do.

## Actual Behavior

ERRO[0002] envconfig.Process: assigning K6_DATADOG_TAG_BLACKLIST to TagBlacklist: converting 'name,url,endpoint' to type stats.TagSet. details: invalid map item: "name"

## Steps to Reproduce the Problem

1. `export K6_DATADOG_TAG_BLACKLIST="name,url,endpoint"`
2.
```bash
K6_DATADOG_ADDR="dogstatsd:8125" k6 run -e HOST='jsonplaceholder.typicode.com' -e ENV= -e TEST_BUILD=3 --vus 1 --duration 10s --summary-trend-stats "min,avg,med,max,p(25),p(75),p(95),p(99)" --out datadog scripts/benchmark/browsing.js
```
3.
```bash
ERRO[0002] envconfig.Process: assigning K6_DATADOG_TAG_BLACKLIST to TagBlacklist: converting 'endpoint' to type stats.TagSet. details: invalid map item: "endpoint"
make: *** [Makefile:65: dd-benchmark] Error 255
```
if I do this `export K6_DATADOG_TAG_BLACKLIST=""` which is the default, all is well, however since Datadog charges per uniqueness, I would like to reduce certain metrics such as:
- tls_version
- name / URL (they are basically the same metrics)
- http

etc.

**Comment by MStoykov** [comment]
Thanks for reporting this @guongle-ssense

Apparently nobody tested that it hasn't been broken around #1148 but with #1604 it seems to work ;)

**Comment by guongle-ssense** [comment]
@MStoykov 
Question:

I have upgraded the k6 to v0.28.0 since the release stated the bug fix is there

however when i run
```
K6_DATADOG_TAG_BLACKLIST="name,url,tls,tls_version,proto,method,image,instance-type,k8s.io/role/node,k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io,k8s.io/cluster-autoscaler/node-template/taint/dedicated:benchmark,k8s.io/cluster-autoscaler/qa.k8s.ssense.com,kernel,kops.k8s.io/instancegroup,kube_node_role,kubernetescluster,pillar,region,security-group,service,team,host,none" K6_DATADOG_ADDR="dogstatsd:8125" k6 run --summary-trend-stats "min,avg,med,max,p(25),p(75),p(95),p(99)" --out datadog scripts/myscript.js
```

When i go to the Datadog Metrics Explorer I still see all the TAG being there.
as Extra, I have also put them inside the ENV (confirmed via `printenv`) 

Am i using it wrong or am i understanding the blacklist operation incorrectly?

**Comment by MStoykov** [comment]
Hi @guongle-ssense, 

Can you screenshot what you see as I can't find something like that? 
Also is this possible that these tags are leftover from your previous tests? Maybe you need to delete them (somehow, not a ... experienced datadog users :( )

From what I can find what you are doing should work, and also we released [v0.29.0](https://github.com/loadimpact/k6/releases/tag/v0.29.0) last week, so you might be interested in upgrading to that as well (no datadog changes there, unfortunately)
